src/qt/frame.cpp

```diff
@@ -31,6 +31,7 @@ class wxQtMainWindow : public wxQtEventSignalHandler< QMainWindow, wxFrame >
     wxQtMainWindow( wxWindow *parent, wxFrame *handler );
 
 private:
+    virtual bool focusNextPrevChild(bool) { return false; }
```

Comment:
Unrelated to the original comment, but it'd be nice to use `wxOVERRIDE` in wxQt too, it's useful as a reminder to update the code if the base class changes later and would also avoid the original question.